types/googlemaps/reference/map.d.ts

```diff
@@ -424,7 +424,7 @@ declare namespace google.maps {
          * set when the map is initialized.
          */
         backgroundColor?: string;
-        /** The initial Map center. Required. */
+        /** The initial Map center. */
```

Comment:
The current description is fully consistent with [the official documentation](https://developers.google.com/maps/documentation/javascript/reference/map#MapOptions.center).

Reply:
I'll fix that. 😄

Reply:
Official documentation updated.